LISISO/install.sh

```diff
@@ -87,6 +81,94 @@ if [[ $distro_version != "5"* ]]; then
  fi
 fi
 
+#Declarations
+
+mlx_driver_size=50
```

Comment:
Is this required ? when we are quering the size of RPM which includes the mlx driver too.